src/octopeer-github/main/Settings/Settings.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * Created by Mitchell on 12-5-2016.
+ */
+
+/**
+ * Class from which user settings can be retrieved.
+ * These settings have been maintained by the front-end.
+ */
+class Settings {
```

Comment:
Like this: [Namespace equivalent of `getInstance()`](http://stackoverflow.com/a/30174360/3227948)